### ISSUE TITLE
Control Z axis for custom dataset

### ISSUE THREAD

**Issue by reddittoob** [traveller59/second.pytorch#454]
Hi, I'm trying to train my customo dataset.

However, when I generate ground truth, my data is cut like below image.

![custom](https://user-images.githubusercontent.com/30588581/115002352-9e8f0200-9edf-11eb-8534-8d98053cde36.png)

Anyone somebody know how control Z axis?

Thanks.

**Comment by young147** [comment]
You can comment at line 239 in kitti_dataset.py,because there is no need for us to convert box center.
